app/assets/javascripts/dashboard/dataset_panel.js

```diff
@@ -0,0 +1,192 @@
+            backgroundSize: "cover",
+            width: "100%",
+            height: "100%",
+          }}
+        />
+      </span>
+      <div className="dataset-description">
+        <div className="description-flex">
+          <h3>{name}</h3>
+          <div className="dataset-description-body">{description}</div>
+        </div>
+      </div>
+    </React.Fragment>
+  );
+}
+
+function ThumbnailAndDescriptionFromDataset({ dataset }: { dataset: DatasetType }) {
+  return (
+    <ThumbnailAndDescription
+      thumbnailURL={dataset.thumbnailURL}
+      name={getDisplayName(dataset)}
+      description={getDescription(dataset)}
+    />
+  );
+}
+
+class DatasetPanel extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      showLessContent: true,
+    };
+  }
+
+  handleClick = () => {
+    this.setState({ showLessContent: !this.state.showLessContent });
+  };
+
+  renderCard = (dataset: DatasetType) => (
+    <a href={`/datasets/${dataset.name}/view`} title="View Dataset">
+      <Card bodyStyle={{ padding: 0 }} className="spotlight-item-card">
+        <ThumbnailAndDescriptionFromDataset dataset={dataset} />
+      </Card>
+    </a>
+  );
+
+  renderMultiDatasetCard = (groupName: string, datasets: DatasetType[]) => {
+    const multiDescription = (
+      <div>
+        This collection consists of multiple datasets:
+        <ul>
+          {datasets.map(dataset => {
+            const popoverContent = (
+              <div className="spotlight-popover-card">
+                <ThumbnailAndDescriptionFromDataset dataset={dataset} />
+              </div>
+            );
+            return (
+              <li key={dataset.name}>
+                <Popover
+                  placement="left"
+                  content={popoverContent}
+                  trigger="hover"
+                  overlayClassName="antd-spotlight-popover-card"
+                >
+                  <a href="#">{getDisplayName(dataset)}</a>
+                </Popover>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+    );
+
+    return (
+      <Card bodyStyle={{ padding: 0 }} className="spotlight-item-card">
+        <ThumbnailAndDescription
+          thumbnailURL={datasets[0].thumbnailURL}
+          name={groupName}
+          description={multiDescription}
+        />
+      </Card>
+    );
+  };
+
+  render() {
+    const groupedDatasets = _.entries(
+      // Instead of dataset.name, this could be grouped by a group tag
+      _.groupBy(this.props.datasets, dataset => dataset.name),
+    );
+    const maybeCroppedDatasetsGroup = groupedDatasets.slice(
+      0,
+      this.state.showLessContent && this.props.limitDefaultDatasetCount
+        ? croppedDatasetCount
+        : this.props.datasets.length,
+    );
+
+    return (
+      <div className="dataset-panel">
+        {this.props.showOrganizationHeader && <h1>{this.props.owningOrganization}</h1>}
```

Comment:
`&&`? why no ternary

Reply:
I tend to use `&&` for the case in which the alternative in the ternary would be `null`. I think it's not too uncommon in react code bases (also in the [official docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)). Do you prefer ternaries?